controlplane/migrations/0097_melodic_the_executioner.sql

```diff
@@ -0,0 +1,77 @@
+DO $$ BEGIN
+ CREATE TYPE "public"."graph_pruning_rules" AS ENUM('UNUSED_FIELDS', 'DEPRECATED_FIELDS', 'FORCE_DEPRECATION_BEFORE_DELETION');
+EXCEPTION
+ WHEN duplicate_object THEN null;
+END $$;
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "field_grace_period" (
+	"id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"subgraph_id" uuid NOT NULL,
+	"organization_id" uuid NOT NULL,
+	"namespace_id" uuid NOT NULL,
+	"path" text,
+	"expires_at" timestamp with time zone NOT NULL,
+	"is_deprecated" boolean
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "namespace_graph_pruning_check_config" (
+	"id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"namespace_id" uuid NOT NULL,
+	"graph_pruning_rule" "graph_pruning_rules" NOT NULL,
+	"severity_level" "lint_severity" NOT NULL,
+	"grace_period" integer NOT NULL,
+	"scheme_usage_check_period" integer,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "schema_check_graph_pruning_action" (
+	"id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"schema_check_id" uuid NOT NULL,
+	"graph_pruning_rule" "graph_pruning_rules" NOT NULL,
+	"field_path" text NOT NULL,
+	"message" text,
+	"is_error" boolean DEFAULT false,
+	"location" json NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"federated_graph_id" uuid NOT NULL
+);
+--> statement-breakpoint
+ALTER TABLE "namespaces" ADD COLUMN "enable_graph_pruning" boolean DEFAULT false NOT NULL;--> statement-breakpoint
```

Comment:
This should not be done on the namespace entity. We manage features differently. We did the mistake with linting already. Can you move it?